packages/base/src/asset-registries/Illustrations.ts

```diff
@@ -36,7 +36,11 @@ const registerIllustrationLoader = (illustrationName: string, loader: Illustrati
 const _loadIllustrationOnce = async (illustrationName: string) => {
 	if (!illustrationPromises.has(illustrationName)) {
 		if (!loaders.has(illustrationName)) {
-			throw new Error(`No loader registered for the ${illustrationName} illustration. Probably you forgot to import the "@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js" module.`);
+			if (illustrationName.startsWith("Tnt")) {
```

Comment:
If you want you can make it shorter

cont illustrationPath = illustrationName.startsWith("Tnt") ? tnt/${illustrationName.replace(/^Tnt/, "") : illustrationName;

throw new Error(`No loader registered for the ${illustrationName} illustration. Probably you forgot to import the "@ui5/webcomponents-fiori/dist/illustrations/${illustrationPath}.js" module. Or you can import all illustrations by importing the "@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js" module.`)